src/api/test/functional/webui/spider_test.rb

```diff
@@ -7,6 +7,28 @@
 ENV['BOOTSTRAP'] = '1'
 
 class Webui::SpiderTest < Webui::IntegrationTest
+  def ignore_link?(link)
+    return 1 if link =~ %r{/mini-profiler-resources}
+    # that link is just a top ref
+    return 1 if link.end_with?('/package/rdiff')
+    # admin can see even the hidden
+    return 1 if link.end_with?('/package/show/HiddenRemoteInstance')
+    return 1 if link =~ %r{/package/show/SourceprotectedProject}
+    # this is crashing (bug)
+    return 1 if link =~ %r{/package/show/UseRemoteInstance}
```

Comment:
Note that this bug is real, but I didn't report it. You can see it in action if you follow any package name on https://build-test.opensuse.org/project/monitor/OBS:Server:Unstable?arch_x86_64=1&broken=1&defaults=0&repo_SLE_12_SP4=1